package.json

```diff
@@ -1,21 +1,21 @@
 {
-  "name": "karma-dojo",
-  "version": "0.0.1",
-  "description": "A Karma plugin. Adapter for Dojo Toolkit.",
+  "name": "karma-dojo-wrapper",
+  "version": "0.0.2",
```

Comment:
Other than declaring dojo as peer dependency, this line (version bump) is pretty much the only other change to this file that is suitable for PR back to this repo.